app/design/frontend/Magento/luma/Magento_Theme/web/css/source/_module.less

```diff
@@ -3,8 +3,6 @@
 //  * See COPYING.txt for license details.
 //  */
 
-@import 'module/_collapsible_navigation.less';
```

Comment:
Now the file won't be imported anymore. This line shouldn't be removed because this `_module.less` (and not an extend) overrides the `_module.less` from Blank theme.